fpgaconvnet/proto/regen.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-protoc --python_out=. fpgaconvnet.proto
+~/.local/bin/protoc --python_out=. fpgaconvnet.proto
```

Comment:
remove this, as this is particular to your environment. You can add protoc to your path.